### PR TITLE
chore: Bump grunt-html

### PR DESCRIPTION
**Pull request by nschonni**
Has some additional error handling and mutes the old "details" error

### CODE REVIEW

**Comment by nschonni** [comment]
Since some of the language checks seem fragile and are likely to give us false positives with our small fragments, I've also submitted https://github.com/jzaefferer/grunt-html/pull/119 to be able to opt out of the language checking

File: Gruntfile.coffee
Comment (EricDunsworth) [inline Gruntfile.coffee]: Gonna have to ignore Dutch too to prevent Travis-CI from failing:
```
"This document appears to be written in Dutch but the “html” start tag has “lang=\"en\"”. Consider using “lang=\"nl\"” (or variant) instead."
```

Or maybe just hold off until jzaefferer/grunt-html#119 is merged.

File: Gruntfile.coffee
Comment (EricDunsworth) [inline Gruntfile.coffee]: FWIW, I tried removing those in #6993 but it was decided to hold off on it to benefit older screen readers. Although that reasoning might not be as relevant anymore in 2018...

**Comment by duboisp** [comment]
@nschonni the upstream PR was merged, what need to be done to finalize this PR?

**Comment by nschonni** [comment]
Waiting for grunt-html to release a new version with the fix included

**Review by duboisp — APPROVED** [review]

**Pull request merged**
